### ISSUE TITLE
when using xinitrc in fedora silverblue toolbox containers fail to connect to x

### ISSUE THREAD

**Issue by yilkalargaw** [containers/toolbox#408]
When I disable gdm display manager in fedora silverblue my toolboxes fail to connect to xorg so all gui apps inside toolboxes fail with a message indication faliure to find display 0.

**Comment by yilkalargaw** [comment]
the same thing happens on regular fedora

**Comment by debarshiray** [comment]
What's the value of the `DISPLAY` environment variable?

Processes inside toolbox containers use an abstract UNIX socket (eg., `@/tmp/.X11-unix/X<n>`) to connect to the X.org server. Are you sure that you aren't running X with something like `-nolisten local`, which is making it not listen on abstract socket?

**Comment by yilkalargaw** [comment]
No, I just use regular fedora silver blue and use the command ```startx /usr/bin/awesome```. I don't really know the internals of the startx command to comment about which socket it uses.
